<ui version="4.0" >
 <author>Aaron Racicot</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>499</height>
   </rect>
  </property>
  <property name="font" >
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <property name="windowIcon" >
   <iconset resource="opusmain.qrc" >
    <normaloff>:/Images/Images/cog.png</normaloff>:/Images/Images/cog.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>54</y>
     <width>802</width>
     <height>425</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" >
    <item>
     <widget class="QSplitter" name="splitter" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="toolBox" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize" >
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="font" >
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="currentIndex" >
        <number>0</number>
       </property>
       <widget class="QWidget" name="generalmanager_page" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>399</width>
          <height>372</height>
         </rect>
        </property>
        <attribute name="title" >
         <string>General</string>
        </attribute>
        <layout class="QGridLayout" />
       </widget>
       <widget class="QWidget" name="datamanager_page" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>399</width>
          <height>372</height>
         </rect>
        </property>
        <attribute name="title" >
         <string>Data</string>
        </attribute>
        <layout class="QGridLayout" >
         <item row="0" column="0" >
          <widget class="QTabWidget" name="dataManager_toolBox" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font" >
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="currentIndex" >
            <number>0</number>
           </property>
           <widget class="QWidget" name="datamanager_xmlconfig" >
            <property name="geometry" >
             <rect>
              <x>0</x>
              <y>0</y>
              <width>369</width>
              <height>325</height>
             </rect>
            </property>
            <attribute name="title" >
             <string>Tools</string>
            </attribute>
            <layout class="QGridLayout" />
           </widget>
           <widget class="QWidget" name="datamanager_dirview" >
            <property name="geometry" >
             <rect>
              <x>0</x>
              <y>0</y>
              <width>372</width>
              <height>325</height>
             </rect>
            </property>
            <attribute name="title" >
             <string>Opus Data</string>
            </attribute>
            <layout class="QGridLayout" />
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="modelmanager_page" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>399</width>
          <height>372</height>
         </rect>
        </property>
        <attribute name="title" >
         <string>Models</string>
        </attribute>
        <layout class="QGridLayout" />
       </widget>
       <widget class="QWidget" name="runmanager_page" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>399</width>
          <height>372</height>
         </rect>
        </property>
        <attribute name="title" >
         <string>Scenarios</string>
        </attribute>
        <layout class="QGridLayout" />
       </widget>
       <widget class="QWidget" name="resultsmanager_page" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>399</width>
          <height>372</height>
         </rect>
        </property>
        <attribute name="title" >
         <string>Results</string>
        </attribute>
        <layout class="QGridLayout" />
       </widget>
      </widget>
      <widget class="QTabWidget" name="tabWidget" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="sizeIncrement" >
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize" >
        <size>
         <width>1000</width>
         <height>0</height>
        </size>
       </property>
       <property name="font" >
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="tabShape" >
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex" >
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_editorView" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>360</width>
          <height>372</height>
         </rect>
        </property>
        <attribute name="title" >
         <string>Editor</string>
        </attribute>
        <attribute name="icon" >
         <iconset resource="opusmain.qrc" >
          <normaloff>:/Images/Images/table_lightning.png</normaloff>:/Images/Images/table_lightning.png</iconset>
        </attribute>
        <layout class="QGridLayout" />
       </widget>
       <widget class="QWidget" name="tab_pythonView" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>360</width>
          <height>372</height>
         </rect>
        </property>
        <attribute name="title" >
         <string>Python Console</string>
        </attribute>
        <attribute name="icon" >
         <iconset resource="opusmain.qrc" >
          <normaloff>:/Images/Images/python_type.png</normaloff>:/Images/Images/python_type.png</iconset>
        </attribute>
        <layout class="QGridLayout" >
         <item row="1" column="0" >
          <widget class="QWidget" native="1" name="pythonLineWidget" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="pythonLabel" >
              <property name="text" >
               <string>>>></string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="pythonLineEdit" />
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0" >
          <widget class="QWidget" native="1" name="pythonWidget" />
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_logView" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>360</width>
          <height>372</height>
         </rect>
        </property>
        <attribute name="title" >
         <string>Log View</string>
        </attribute>
        <attribute name="icon" >
         <iconset resource="opusmain.qrc" >
          <normaloff>:/Images/Images/table.png</normaloff>:/Images/Images/table.png</iconset>
        </attribute>
        <layout class="QGridLayout" >
         <item row="0" column="0" >
          <widget class="QTextBrowser" name="logViewTextBrowser" />
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProject" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actOpenProject" />
    <addaction name="separator" />
    <addaction name="actSaveProject" />
    <addaction name="actSaveProjectAs" />
    <addaction name="actCloseProject" />
    <addaction name="separator" />
    <addaction name="actExit" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="actAbout" />
   </widget>
   <widget class="QMenu" name="menuUtilities" >
    <property name="title" >
     <string>Tools</string>
    </property>
    <addaction name="actPythonView" />
    <addaction name="actLogView" />
    <addaction name="actEditorView" />
    <addaction name="separator" />
    <addaction name="actVariableLibrary" />
    <addaction name="actLaunchResultBrowser" />
    <addaction name="separator" />
    <addaction name="actPreferences" />
    <addaction name="actDatabaseSettings" />
   </widget>
   <addaction name="menuProject" />
   <addaction name="menuUtilities" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>479</y>
     <width>802</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>22</y>
     <width>802</width>
     <height>32</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <property name="iconSize" >
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="toolButtonStyle" >
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actOpenProject" />
   <addaction name="actSaveProject" />
   <addaction name="actCloseProject" />
   <addaction name="separator" />
   <addaction name="actVariableLibrary" />
   <addaction name="actLaunchResultBrowser" />
   <addaction name="separator" />
   <addaction name="actLogView" />
   <addaction name="separator" />
   <addaction name="actDatabaseSettings" />
   <addaction name="separator" />
   <addaction name="actPreferences" />
  </widget>
  <action name="mpActionZoomOut" >
   <property name="icon" >
    <iconset resource="opusmain.qrc" >
     <normaloff>:/Images/Images/mActionZoomOut.png</normaloff>:/Images/Images/mActionZoomOut.png</iconset>
   </property>
   <property name="text" >
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="mpActionPan" >
   <property name="icon" >
    <iconset resource="opusmain.qrc" >
     <normaloff>:/Images/Images/mActionPan.png</normaloff>:/Images/Images/mActionPan.png</iconset>
   </property>
   <property name="text" >
    <string>Pan</string>
   </property>
  </action>
  <action name="mpActionAddRasterLayer" >
   <property name="icon" >
    <iconset resource="opusmain.qrc" >
     <normaloff>:/Images/Images/mActionAddRasterLayer.png</normaloff>:/Images/Images/mActionAddRasterLayer.png</iconset>
   </property>
   <property name="text" >
    <string>Add Raster Layer</string>
   </property>
  </action>
  <action name="mpActionZoomIn" >
   <property name="icon" >
    <iconset resource="opusmain.qrc" >
     <normaloff>:/Images/Images/mActionZoomIn.png</normaloff>:/Images/Images/mActionZoomIn.png</iconset>
   </property>
   <property name="text" >
    <string>Zoom In</string>
   </property>
  </action>
  <action name="mpActionAddVectorLayer" >
   <property name="icon" >
    <iconset resource="opusmain.qrc" >
     <normaloff>:/Images/Images/mActionAddLayer.png</normaloff>:/Images/Images/mActionAddLayer.png</iconset>
   </property>
   <property name="text" >
    <string>Add Vector Layer</string>
   </property>
  </action>
  <action name="actionOpen_Project" >
   <property name="icon" >
    <iconset resource="opusmain.qrc" >
     <normaloff>:/Images/Images/folder.png</normaloff>:/Images/Images/folder.png</iconset>
   </property>
   <property name="text" >
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionSave_Project" >
   <property name="icon" >
    <iconset resource="opusmain.qrc" >
     <normaloff>:/Images/Images/script_save.png</normaloff>:/Images/Images/script_save.png</iconset>
   </property>
   <property name="text" >
    <string>Save Project</string>
   </property>
  </action>
  <action name="actionSave_Project_As" >
   <property name="icon" >
    <iconset resource="opusmain.qrc" >
     <normaloff>:/Images/Images/script_save.png</normaloff>:/Images/Images/script_save.png</iconset>
   </property>
   <property name="text" >
    <string>Save Project As...</string>
   </property>
  </action>
  <action name="actOpenProject" >
   <property name="icon" >
    <iconset resource="opusmain.qrc" >
     <normaloff>:/Images/Images/folder_database.png</normaloff>:/Images/Images/folder_database.png</iconset>
   </property>
   <property name="text" >
    <string>Open Project</string>
   </property>
  </action>
  <action name="actExit" >
   <property name="icon" >
    <iconset resource="opusmain.qrc" >
     <normaloff>:/Images/Images/cross.png</normaloff>:/Images/Images/cross.png</iconset>
   </property>
   <property name="text" >
    <string>Exit</string>
   </property>
  </action>
  <action name="actSaveProject" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="opusmain.qrc" >
     <normaloff>:/Images/Images/disk.png</normaloff>:/Images/Images/disk.png</iconset>
   </property>
   <property name="text" >
    <string>Save Project</string>
   </property>
  </action>
  <action name="actSaveProjectAs" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="opusmain.qrc" >
     <normaloff>:/Images/Images/script_save.png</normaloff>:/Images/Images/script_save.png</iconset>
   </property>
   <property name="text" >
    <string>Save Project As...</string>
   </property>
  </action>
  <action name="actionRun_Manager" >
   <property name="icon" >
    <iconset resource="opusmain.qrc" >
     <normaloff>:/Images/Images/cog.png</normaloff>:/Images/Images/cog.png</iconset>
   </property>
   <property name="text" >
    <string>Run Manager</string>
   </property>
  </action>
  <action name="actAbout" >
   <property name="icon" >
    <iconset resource="opusmain.qrc" >
     <normaloff>:/Images/Images/help.png</normaloff>:/Images/Images/help.png</iconset>
   </property>
   <property name="text" >
    <string>About</string>
   </property>
  </action>
  <action name="tabActionEditor" >
   <property name="icon" >
    <iconset resource="opusmain.qrc" >
     <normaloff>:/Images/Images/table.png</normaloff>:/Images/Images/table.png</iconset>
   </property>
   <property name="text" >
    <string>Editor</string>
   </property>
  </action>
  <action name="tabActionMapView" >
   <property name="icon" >
    <iconset resource="opusmain.qrc" >
     <normaloff>:/Images/Images/map.png</normaloff>:/Images/Images/map.png</iconset>
   </property>
   <property name="text" >
    <string>Map View</string>
   </property>
  </action>
  <action name="tabActionPythonConsole" >
   <property name="icon" >
    <iconset resource="opusmain.qrc" >
     <normaloff>:/Images/Images/python_type.png</normaloff>:/Images/Images/python_type.png</iconset>
   </property>
   <property name="text" >
    <string>Python Console</string>
   </property>
  </action>
  <action name="tabActionLogView" >
   <property name="icon" >
    <iconset resource="opusmain.qrc" >
     <normaloff>:/Images/Images/folder.png</normaloff>:/Images/Images/folder.png</iconset>
   </property>
   <property name="text" >
    <string>Log View</string>
   </property>
  </action>
  <action name="actPythonView" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="opusmain.qrc" >
     <normaloff>:/Images/Images/python_type.png</normaloff>:/Images/Images/python_type.png</iconset>
   </property>
   <property name="text" >
    <string>Python Console</string>
   </property>
   <property name="toolTip" >
    <string>Open Python Console</string>
   </property>
   <property name="visible" >
    <bool>true</bool>
   </property>
   <property name="iconVisibleInMenu" >
    <bool>true</bool>
   </property>
  </action>
  <action name="actLogView" >
   <property name="icon" >
    <iconset resource="opusmain.qrc" >
     <normaloff>:/Images/Images/table.png</normaloff>:/Images/Images/table.png</iconset>
   </property>
   <property name="text" >
    <string>Log View</string>
   </property>
   <property name="toolTip" >
    <string>Open Log View</string>
   </property>
  </action>
  <action name="actEditorView" >
   <property name="icon" >
    <iconset resource="opusmain.qrc" >
     <normaloff>:/Images/Images/table_lightning.png</normaloff>:/Images/Images/table_lightning.png</iconset>
   </property>
   <property name="text" >
    <string>Editor View</string>
   </property>
   <property name="toolTip" >
    <string>Open Editor View</string>
   </property>
  </action>
  <action name="actionMap_View" >
   <property name="icon" >
    <iconset resource="opusmain.qrc" >
     <normaloff>:/Images/Images/map.png</normaloff>:/Images/Images/map.png</iconset>
   </property>
   <property name="text" >
    <string>Map View</string>
   </property>
  </action>
  <action name="actionIncreaseFontSize" >
   <property name="text" >
    <string>Increase Font Size</string>
   </property>
  </action>
  <action name="actionDecreaseFontSize" >
   <property name="text" >
    <string>Decrease Font Size</string>
   </property>
  </action>
  <action name="actCloseProject" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="opusmain.qrc" >
     <normaloff>:/Images/Images/folder.png</normaloff>:/Images/Images/folder.png</iconset>
   </property>
   <property name="text" >
    <string>Close Project</string>
   </property>
   <property name="toolTip" >
    <string>Close Project</string>
   </property>
  </action>
  <action name="actVariableLibrary" >
   <property name="icon" >
    <iconset resource="opusmain.qrc" >
     <normaloff>:/Images/Images/book_edit.png</normaloff>:/Images/Images/book_edit.png</iconset>
   </property>
   <property name="text" >
    <string>Variable Library</string>
   </property>
   <property name="toolTip" >
    <string>Open Variable Library</string>
   </property>
  </action>
  <action name="actPreferences" >
   <property name="icon" >
    <iconset resource="opusmain.qrc" >
     <normaloff>:/Images/Images/script.png</normaloff>:/Images/Images/script.png</iconset>
   </property>
   <property name="text" >
    <string>Preferences</string>
   </property>
   <property name="toolTip" >
    <string>Open Preferences</string>
   </property>
  </action>
  <action name="actLaunchResultBrowser" >
   <property name="icon" >
    <iconset resource="opusmain.qrc" >
     <normaloff>:/Images/Images/layout.png</normaloff>:/Images/Images/layout.png</iconset>
   </property>
   <property name="text" >
    <string>Result Browser</string>
   </property>
   <property name="toolTip" >
    <string>Open Result Browser</string>
   </property>
  </action>
  <action name="actDatabaseSettings" >
   <property name="icon" >
    <iconset resource="opusmain.qrc" >
     <normaloff>:/Images/Images/database_link.png</normaloff>:/Images/Images/database_link.png</iconset>
   </property>
   <property name="text" >
    <string>Database Connection Settings</string>
   </property>
   <property name="toolTip" >
    <string>Open Database Connection Settings</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="opusmain.qrc" />
 </resources>
 <connections/>
</ui>
